File: loan/src/main/java/io/github/susimsek/loan/LoanApplication.java
package io.github.susimsek.loan;

import io.github.susimsek.loan.dto.LoanContactInfoDTO;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

@EnableDiscoveryClient
@EnableConfigurationProperties(value = LoanContactInfoDTO.class)
@SpringBootApplication
public class LoanApplication {

    public static void main(String[] args) {
        SpringApplication.run(LoanApplication.class, args);
    }

}

File: loan/src/main/kotlin/io/github/susimsek/loan/LoanApplication.kt
package io.github.susimsek.loan

import io.github.susimsek.loan.dto.LoanContactInfoDTO
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@EnableDiscoveryClient
@EnableConfigurationProperties(value = [LoanContactInfoDTO::class])
@SpringBootApplication
class LoanApplication
fun main(args: Array<String>) {
    runApplication<LoanApplication>(*args)
}
